src/Mod/Import/App/AppImportPy.cpp

```diff
@@ -243,14 +238,10 @@ class Module : public Py::ExtensionModule<Module>
             if (mode >= 0)
                 ocaf.setMode(mode);
             ocaf.loadShapes();
-#elif 1
+

```

Comment:
The code inside the `#elif 1` must also be removed as otherwise it causes a build failure.